index.html

```diff
@@ -915,37 +989,124 @@ <h4>Text</h4>
+            <li>Otherwise (if it is not empty):
+              <ul>
+                <li>if its value is the same as the language of the <a>Text</a> object,
+                  the <a>Text</a> is <a>Original</a>.</li>
+                <li>otherwise (if the value is different to the language of the <a>Text</a>),
+                  the <a>Text</a> is a <a>Translation</a>,
+                  whose source language is the <a>Text Language Source</a> property's value.</li>
+              </ul>
             </li>
-            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of the Original language text in a different language.</li>
           </ul>
-          <aside class="note">The language of a <a>Text</a> object whose <a>Text Language Source</a> is <a>Original</a>
-            can be different to the document's <a>Primary Language</a>.
-            For example a <a>transcript</a> of a short piece of dialogue spoken in a different language to the majority of the dialogue would be annotated
-            with both a different <code>xml:lang</code> value and as being <a>Original</a>.</aside>
-          <p>The <a>Text Language Source</a> property is represented in a <a>DAPT Document</a> by a <code>daptm:langSrc</code> attribute with the following constraints:</p>
-          <p class="ednote">Should we use an abbreviated attribute name?</p>
-          <p class="ednote">Initial design is to use an abbreviated name and <code>original|translation</code>,
-            though I considered using an abbreviated value too, since this attribute will appear on every <code>&lt;p&gt;</code> element.
-            Abbreviating O for Original is probably a bad idea because the letter O and the number 0 can easily be confused.
-            I also considered P for Primary but that caused potential confusion between Primary Language and Primary Text Language Source.</p>
+          <p><a>Text Language Source</a> is an inheritable property.</p>
+          <p>The <a>Text Language Source</a> property is represented in a <a>DAPT Document</a> by a <dfn><code>daptm:langSrc</code></dfn> attribute
+            with the following syntax, constraints and semantics:</p>
           <div class="exampleInner">
             <pre class="language-abnf">
 daptm:langSrc
-  : "original"
-  | "translation"
+: &lt;empty-string&gt; | &lt;language-identifier&gt;
+
+&lt;empty-string&gt;
+: ""                    # default
+
+&lt;language-identifier&gt;   # valid BCP-47 language tag
             </pre>
           </div>
+          <ul>
+            <li>The value MUST be an empty string or a language identifier as defined by [[BCP47]].</li>
+            <li>The default value is the empty string.</li>
+            <li>It <em>applies</em> to <code>&lt;p&gt;</code> and <code>&lt;span&gt;</code> elements.</li>
+            <li>It MAY be specified on the following elements: <code>&lt;tt&gt;</code>, <code>&lt;p&gt;</code> and <code>&lt;span&gt;</code>.</li>
```

Comment:
I'm still wondering about `div` and `body`.
On the one hand, I understand your argument about matching the data model. On the other hand, aligning with `xml:lang` is pleasant.

Reply:
Me too - I'm slightly uncomfortable missing `div` and `body` out, but I'm also struggling to understand the use case for including them, other than completeness. This is my "compromise" position between @andreastai's "only on `p` and `span`, and require it" position and my "provide a scheme where omitting it makes sense, especially for audio description". If we find a need to add `div` and `body` back in later, we could do that.

Reply:
Thanks, @nigelmegitt for the effort. I think this edit is an important improvement. You only need to check in two places to get the value of the source language.